lib/msal-node/test/config/ClientConfiguration.spec.ts

```diff
@@ -69,6 +69,17 @@ describe('ClientConfiguration tests', () => {
         expect(config.auth!.clientId).toEqual(TEST_CONSTANTS.CLIENT_ID);
     });
 
+    test('builds configuration and assigns default authority when empty string provided', () => {
+        const config: Configuration = buildAppConfiguration({
+            auth: {
+                clientId: TEST_CONSTANTS.CLIENT_ID,
+                authority: " "
```

Comment:
Do we have a test case for `authority=""`?

Reply:
Just added this.

Reply:
In common, we have this logic for empty string: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/8c7c0ea26f4f42078e2ef62cebe9a3f6c88dd261/lib/msal-common/src/authority/AuthorityFactory.ts#L50

It seems like today, empty string is treated as invalid use case, as we throw an error. Seems like we should be consistent between all of our libraries about this?

Reply:
In other words, it seems like the change to fallback to the default authority should be done in msal-common, as opposed to in MSAL Node?